tests/e2e/pageobjects/ide/TopMenu.ts

```diff
@@ -46,7 +44,6 @@ export class TopMenu {
 
         const buttonLocator: By = this.getTopMenuButtonLocator(buttonText);
 
-        await this.ide.closeAllNotifications();
```

Comment:
@Ohrimenko1988 any reason why this one was removed?

Reply:
As far as I see this is a redundant step which were implemented in the times when notification was displaying at the top of the screen. And notification could overlap the top menu. Currently it is impossible so no reasons to keep this step. Besides it leads to errors from time to time.

Reply:
sounds reasonable 👍